.github/actions/build/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Setup
+description: "Setup Project"
+runs:
+  using: "composite"
+  steps:
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+      shell: bash
+
+    - uses: actions/cache@v3
+      id: yarn-cache
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+
+    - uses: actions/cache@v3
+      id: turbo-cache
+      with:
+        path: .turbo-cache
+        key: turbo-${{ github.sha }}
+        restore-keys: |
+          turbo-
```

Comment:
curious how these keys/prefixes work? Any docs you can link to?

Reply:
`key` is the id of the cache that will be created when the workflow completes with the contents of path.
`restore-keys` is a list of prefixes to restore from. If a cache key matches a restore prefix, it will be loaded. If there are multiple matches, the most recent one will be pulled.

https://github.com/actions/cache